pkg/component/controller/cplb/cplb_linux.go

```diff
@@ -426,7 +436,11 @@ func (k *Keepalived) redirectToProxyIPTables(op string) error {
 				k.log.Infof("Deleting iptables rule to redirect %s", vip)
 			}
 
-			cmd := exec.Command(filepath.Join(k.K0sVars.BinDir, "iptables"), cmdArgs...)
+			iptablesBin := "iptables"
+			if !k.PrimaryAFIPv4 {
```

Comment:
Wouldn't it make more sense to check if `vip` is an IPv4 or an IPv6 address?

Reply:
Indeed, you found a bug in dual stack clusters